Automatic Validation ended with:
> 2024-04-23T11:10:21.9505354Z ##[error]Manifest Validation Failed
 2024-04-23T11:10:27.0085374Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

--- a/manifests/b/BellSoft/LibericaJDK/8/8.0.412.9/BellSoft.LibericaJDK.8.installer.yaml
+++ b/manifests/b/BellSoft/LibericaJDK/8/8.0.412.9/BellSoft.LibericaJDK.8.installer.yaml
@@ -12,10 +12,10 @@ Installers:
   AppsAndFeaturesEntries:
   - DisplayName: Liberica JDK 8 (64-bit)
 - Architecture: x86
-InstallerUrl: https://download.bell-sw.com/java/8u412+9/bellsoft-jdk8u412+9-windows-i586.msi
-InstallerSha256: 022BA1972448BB007F7917354859193B89A9D861EF9232013FADFC62C68D6E45
-ProductCode: '{A78F5557-4104-4335-94B8-DE599E3523AA}'
-AppsAndFeaturesEntries:
-- DisplayName: Liberica JDK 8 (32-bit)
+  InstallerUrl: https://download.bell-sw.com/java/8u412+9/bellsoft-jdk8u412+9-windows-i586.msi
+  InstallerSha256: 022BA1972448BB007F7917354859193B89A9D861EF9232013FADFC62C68D6E45
+  ProductCode: '{A78F5557-4104-4335-94B8-DE599E3523AA}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liberica JDK 8 (32-bit)
 ManifestType: installer
 ManifestVersion: 1.6.0

